chapters/infrastructure/processing/reading/README.md

```diff
@@ -1,12 +1,163 @@
+
+The recommended usage of the Open Education Builder is through the provided Docker container.
+
+In this sense, one would extend the provided container by creating a new Dockerfile and using the following line at the beginning:
+```Dockerfile
+FROM ghcr.io/open-education-hub/openedu-builder:0.5.1
+```
+Following, any additional dependencies (e.g. Docusaurus, reveal-md) need to be installed.
+```Dockerfile
+# Install node LTS (16)
+RUN curl -fsSL https://deb.nodesource.com/setup_lts.x | bash - && \
+    apt-get update && \
+    apt-get install -yqq nodejs
+
+# Install reveal-md using npm.
+RUN npm install -g reveal-md
+
+# Install Docusaurus.
+RUN npm install create-docusaurus@2.1.0
+```
+
+A [sample Dockerfile](https://github.com/open-education-hub/oer-template/blob/main/Dockerfile) that installs the technologies we recommend is provided in the [`oer-template`](https://github.com/open-education-hub/oer-template) repository.
+
+Additionally, we also provide a Github Action that rebuilds and runs the Docker container and publishes the resulting website whenever content is modified, [here](https://github.com/open-education-hub/oer-template/blob/main/.github/workflows/deployment.yml).
+
+The behavior of the Open Education Builder is controlled through a configuration file that has to be named `config.yaml` and placed at the root of your repository.
+
+Based on the steps defined in the configuration file, the builder will construct plugins and run them in the order specified.
+Each step has an associated input directory and output directory.
+By default, the input directory is the root of the project and the output directory is a directory with the same name under the build directory.
+In the snippet below, there are 3 steps that will run the `command`, `reveal_embed` and `docusaurus` plugins, in that order, with their respective default input and output directories.
+
+```yaml
+stages:
+  - make_assets
+  - embed_reveal
+  - docusaurus
+
+build_dir: /build
+output_dir: /output
+
+make_assets:
+  plugin: command
+# [...]
+
+embed_reveal:
+  plugin: reveal_embed
+# [...]
+
+docusaurus:
+  plugin: docusaurus
+# [...]
+```
+
+Using content that was built/generated in a previous step can be done by using the full path to that content.
+In the example below, we are using content generated in the `make_assets` step as an argument to the `static_assets` option of the `docusaurus` plugin.
+
+```yaml
+make_assets:
+  plugin: command
+  options:
+    command: make
+    locations:
+      - content/chapters/data/lecture
+      - content/chapters/io/lecture
+      # [...]
+    args:
+      - all
+
+# [...]
+
+docusaurus:
+  plugin: docusaurus
+  options:
+# [...]
+    static_assets:
+      - slides/Data: /build/make_assets/content/chapters/data/lecture/_site
+      - slides/IO: /build/make_assets/content/chapters/io/lecture/_site
+# [...]
+
+```
+
+#### Docusaurus
+
+Docusaurus requires a directory structure similar to this:
+```
+docs
+└── Labs
```

Comment:
We avoided concrete examples so far and used either mocks or the methodology repo itself. Not sure if this should also be the case here.

Reply:
I think this is fine. We do aim to use real examples when we have them.